pyproject.toml

```diff
@@ -10,6 +10,9 @@ dependencies = [
     "pydantic>=2.5.3,<3.0.0",
     "typer>=0.9.0",
     "python-dotenv>=1.0.1",
+    "pydantic-ai-slim[openai]>=0.0.8",
```

Comment:
Want to make sure these dont leak into the parent pyproject

Reply:
removed!